core/src/main/java/de/evoila/cf/broker/service/impl/BindingServiceImpl.java

```diff
@@ -128,31 +136,46 @@ private ServiceInstance getBinding(String bindingId) throws ServerviceInstanceBi
 
 
 	protected ServiceInstanceBinding bindServiceKey(String bindingId, ServiceInstance serviceInstance, Plan plan,
-			List<ServerAddress> externalAddresses) throws ServiceBrokerException {
+													List<ServerAddress> externalAddresses, ServiceInstanceBindingRequest request) throws ServiceBrokerException {
 
 		log.debug("bind service key");
 
-		Map<String, Object> credentials = createCredentials(bindingId, serviceInstance, externalAddresses.get(0));
-
+		Map<String, Object> credentials = createCredentials(bindingId, serviceInstance, externalAddresses.get(0), plan, request);
+		List<VolumeMounts> volumeMounts = createMountPoint(bindingId, serviceInstance, externalAddresses.get(0), plan, request, credentials);
 		ServiceInstanceBinding serviceInstanceBinding = new ServiceInstanceBinding(bindingId, serviceInstance.getId(),
```

Comment:
Should this be optional? With this implementation we create a volumeMount for each service broker